src/server.js

```diff
@@ -170,103 +174,195 @@ api.post('/getChunkedData', (req, res) => {
   }
   console.log(`dataPath = ${dataPath}`);
 
-  // call 'vg chunk' to generate graph
-  let vgChunkParams = ['chunk', '-x', `${dataPath}${xgFile}`];
-  if (req.withGam) {
-    // Use a GAM index
-    vgChunkParams.push('-a', `${dataPath}${gamFile}`, '-g');
+  // parse region
+  // either a range -> seq:start-end
+  // or a position and distance -> seq:post+distance
+  let region = req.body.region;
+  let region_col = region.split(":");
+  let start_end = region_col[1].split("-");
+  let pos_dist = region_col[1].split("+");
```

Comment:
This is just going to throw an error on the server if the client's requested region doesn't have ":". We need to actually check for it I think.

Reply:
Good point, I've added checks in the latest comment. I've also added a way to show an error message on the front end for multiple errors, including this one.